package/gluon-status-page/files/lib/gluon/status-page/www/cgi-bin/dyn/neighbours-nodeinfo

```diff
@@ -6,6 +6,6 @@ badrequest() {
         exit 1
 }
 
-( batctl if | cut -d: -f1 | grep -qxF "$QUERY_STRING" ) 2>/dev/null || badrequest
+( ubus call network.interface dump | jsonfilter -e "@.interface[@.proto='gluon_mesh' && @.up=true].device"| grep -qxF "$QUERY_STRING" ) 2>/dev/null || badrequest
```

Comment:
Hmm, I think the complexity of this command is high enough that we should encapsulate the whole thing up to the `||` in a small script provided by gluon-core. Something like `gluon-is-meshif`. I guess the `grep` could also be replaced by an extension of the jsonfilter expression.

Reply:
agreed - how about we create gluon-get-mesh-ifs  that contains everything up to the Igrep. This would be usable in a more universal way than when incorporating the grep for a specific interface....

Reply:
done